tests/e2e/performance/test_pvc_attachtime.py

```diff
@@ -224,6 +226,10 @@ def test_pod_start_time(self, pod_obj_list):
             "test_time", {"start": self.test_start_time, "end": self.test_end_time}
         )
 
+        self.full_results.add_key("interface", self.interface)
```

Comment:
since you gave the 'storageclass' you do no need this, it is duplicate.

Reply:
Done